### DISCUSSION TITLE
How to load a saved .m3u list?

### DISCUSSION THREAD

**Discussion by GaugeAndGravity** [tramhao/termusic#387] [Q&A] [answered]
Hi, maybe I'm too stupid, but I didn't find a way to load a saved list... Can I do that?

**Comment by hasezoey** [comment, marked as answer]
termusic *should* just load a m3u playlist, assuming the files exist, though i have just now discovered that if the files in the playlist arent absolute paths, termusic fails to find those files (and does not print any kind of error or warning).

Edit: a fix is in #393
